client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderLine.ts

```diff
@@ -125,9 +129,28 @@ export function usePurchaseOrderLine(id?: string | null) {
       unit: draft.unit,
       supplierItemCode: setNullableInput('supplierItemCode', draft),
       comment: setNullableInput('comment', draft),
+      status: draft.status,
     };
+    await updatePurchaseOrderLine(input);
     resetDraft();
-    return await updatePurchaseOrderLine(input);
+    return;
+  };
+
+  const updateLineStatus = async (
```

Comment:
Would be nice to move this to its own endpoint in the backend that just passes in ids~... though there are a few places where we're making mass api calls to delete....